src/_11ty/collections/primary.js

```diff
@@ -3,6 +3,9 @@
 module.exports = function (collectionApi) {
   return collectionApi.getAll().filter(item => {
     const { tags = [] } = item.data;
-    return !tags.includes('support');
+
+    const excludeTags = ['support', 'warning'];
```

Comment:
Thought: Would perhaps refer to this as `error` instead of `warning`? I.e. Could be generalised as covering [error pages](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404#custom_error_pages)

Reply:
Ha, I originally had it as `error` but then changed it to `warning`. I'll change it back to `error`.